azure_arc_servers_jumpstart/docs/arc_securitycenter.md

```diff
@@ -0,0 +1,140 @@
+    Output should look like this:
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+  **Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).
+
+# Onboarding Azure Arc Machines on Azure Security Center
+
+* Data collected by Security Center is stored in a Log Analytics workspace, you can either use the default one created by ASC or a custom one created by you. If you want to create a dedicated workspace, you can automate the deployment by editing the ARM template [parameters file](../securitycenter/arm/log_analytics-template.parameters.json), provide a name and location for your workspace:
+
+    ![](../img/securitycenter/03.png)
+
+To deploy the ARM template, navigate to the [deployment folder](../securitycenter/arm) and run the below command:
+
+  ```bash
+    az deployment group create --resource-group <Name of the Azure Resource Group> \
+    --template-file <The *log_analytics-template.json* template file location> \
+    --parameters <The *log_analytics-template.parameters.json* template file location>
+  ```
+
+* If you are going for an user-defined workspace, you should instruct Security Center to use it instead of the default one, use the below command:
+
+  ```bash
+    az security workspace-setting create --name default \
+    --target-workspace '/subscriptions/<Your subscription ID>/resourceGroups/<Name of the Azure Resource Group>/providers/Microsoft.OperationalInsights/workspaces/<Name of the Log Analytics Workspace>'
+  ```
+
+* Select the Azure Security Center tier, the Free tier is enabled on all your Azure subscriptions by default and will provide continuous security assessment and actionable security recommendations. In this guide, you will use the Standard tier for Virtual Machines that extends these capabilities providing unified security management and threat protection across your hybrid cloud workloads. To enable the Standard tier of Azure Security Center for VMs run the command below: 
+
+    ```bash
+    az security pricing create -n VirtualMachines --tier 'standard'
+    ```
+* Now you need to assign the default Security Center policy initiative. ASC makes its security recommendations based on policies, there is an specific initiative that groups Security Center policies with the definition ID '1f3afdf9-d0c9-4c3d-847f-89da613e70a8'. The command below will assign the ASC initiative to your subscription:
+
+    ```bash
+    az policy assignment create --name 'ASC Default <Your subscription ID>' \
+    --scope '/subscriptions/<Your subscription ID>' \
+    --policy-set-definition '1f3afdf9-d0c9-4c3d-847f-89da613e70a8' 
+    ```
```

Comment:
IMO, I think the Log Analytics workspace should be it's own section and not a prereq.